### PR TITLE
feat: adding json schema to chart

### PR DESCRIPTION
**Pull request by mojtabaimani**

### CODE REVIEW

**Review by Morriz — COMMENTED** [review]
Are you asking for a review? Is this in draft? I don't see the json schema (solution) yet. Or is this a new PR for master?

File: chart/otomi/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
```
Comment (j-zimnowoda) [inline chart/otomi/Chart.yaml:1]: nice

File: chart/otomi/templates/job.yaml [hunk 2]
```diff
@@ -20,8 +20,8 @@ spec:
       initContainers:
         - name: bootstrap-values
           securityContext: {{- toYaml .Values.securityContext | nindent 12 }}
-          image: otomi/core:{{ .Values.otomi.version }}
-          imagePullPolicy: Always
+          image: "{{ .Values.coreImage.repository | default "otomi/core" }}:{{ .Values.coreImage.tag | default .Values.otomi.version }}"
```
Comment (j-zimnowoda) [inline chart/otomi/templates/job.yaml:23]: It is a bit inconsistent that you specify default for `image` here and default for `imagePullPolicy` in values.yaml.
Could we follow one pattern?
Now I am also wondering if the `otomi` object should contain imagePullPolicy, repository and version. What do you think?

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
Nice improvements but I do not see code for converting and injecting schema to the chart while releasing - a step in Github workflows. Please take a look on my comment as well.

File: chart/otomi/templates/job.yaml (hunk 2 above)
Comment (Morriz) [reply chart/otomi/templates/job.yaml:23]: just follow the helm chart paradigm

Comment (Morriz) [reply chart/otomi/templates/job.yaml:23]: if that exists, but being pragmatical I mean to look at `helm create xx` and inspect `xx`

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**